### PR TITLE
Cleaning up dead code and removing relocated project

### PR DESCRIPTION
**Pull request by shoatman**

### CODE REVIEW

File: testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java [hunk 1]
```diff
@@ -101,13 +101,14 @@ void handleInteractiveRequestRedirect(int requestCode, int resultCode, Intent da
     // The app does nothing with the newly acquired scopes yet.
     void doExtraScopeRequest(final Activity activity) {
         getUsers();
+        mApplication.acquireToken()
```
Comment (iambmelt) [inline testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java]: Need to delete -- not a statement (no semicolon, no args)

File: testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java [hunk 2]
```diff
@@ -101,13 +101,14 @@ void handleInteractiveRequestRedirect(int requestCode, int resultCode, Intent da
...
-                mUsers.get(0), // The user object
+                "",
                 UiBehavior.CONSENT,
                 "", // Extra query parameters
                 EXTRA_SCOPES,
-                "", // AuthorityMetadata
+                null,
                 new AuthenticationCallback() {
                     @Override
                     public void onSuccess(AuthenticationResult authenticationResult) {
```
Comment (iambmelt) [inline testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java]: Can't see it in the diff, but line `134` contains a call to `getUsers()` -- needs to be removed or adapted to use `getAccount()`

File: msal/src/main/java/com/microsoft/identity/client/internal/controllers/MSALApiDispatcher.java [hunk 3]
```diff
@@ -25,10 +25,10 @@
 import android.content.Intent;
```
Comment (iambmelt) [inline msal/src/main/java/com/microsoft/identity/client/internal/controllers/MSALApiDispatcher.java]: This class appears to have the wrong package name declared above ^^

**Comment by iambmelt** [comment]
@shoatman @addev-ashish Proposed changes in #356

File: testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java (hunk 2 above)
Comment (shoatman) [reply testapps/sample/src/main/java/com/microsoft/identity/client/sample/AuthUtil.java]: Corrected in the PR you sent.

**Comment by iambmelt** [comment]
🎉 🎈 🕺

**Pull request merged**